src/modules/cmdpal/Microsoft.CmdPal.UI/Controls/ShortcutControl/ShortcutWithTextLabelControl.xaml

```diff
@@ -0,0 +1,45 @@
+﻿<UserControl
+    x:Class="Microsoft.CmdPal.UI.Controls.ShortcutWithTextLabelControl"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:controls="using:Microsoft.CmdPal.UI.Controls"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:tk7controls="using:CommunityToolkit.WinUI.UI.Controls"
```

Comment:
tk7?

Reply:
:shrug:
this code was pilfered from `.\src\settings-ui\Settings.UI\SettingsXAML\Controls\ShortcutControl\ShortcutWithTextLabelControl.xaml` and that file had it